docs/visualize.asciidoc

```diff
@@ -1,2 +1,131 @@
 [[visualize]]
-== Visualizing your Data
\ No newline at end of file
+== Visualize
+
+You can use Kibana's _Visualize_ app to design and create data visualizations. You can save these visualizations, use
+them individually, or combine visualizations into a _dashboard_. A visualization can be based on one of the following 
+data source types:
+
+* A new interactive search
+* A saved search
+* An existing saved visualization
+
+Visualizations are based on the {ref}/search-aggregations.html[aggregation] feature introduced in Elasticsearch 1. 
+
+[float]
+[[getting-started]]
+=== Creating a New Visualization
+
+To start the New Visualization wizard, click on the *Visualize* tab at the top left of the screen. Alternately, you can 
+click the *New Visualization* button image:images/K4NewDocument.png[New Document button] in the toolbar panel to the 
+right of the search bar. The wizard guides you through the following steps:
+
+[float]
+[[newvis01]]
+==== Step 1: Choose the Visualization Type
+
```

Comment:
Maybe something about the types of visualizations you can choose from & what they can be used from?